graphql_compiler/tests/test_compiler.py

```diff
@@ -1729,8 +1729,10 @@ def test_fold_and_traverse(self):
                             .collect{
                                 entry -> entry.outV.next()
                             }
-                            .collectMany{entry -> entry.out_Animal_ParentOf.collect{
-                                edge -> edge.inV.next()}
+                            .collectMany{entry -> entry.out_Animal_ParentOf
```

Comment:
There are many more instances of this formatting that need fixing, please do it everywhere.